lib/Basic/AnyValue.cpp

```diff
@@ -0,0 +1,20 @@
+//===--- AnyValue.cpp - Out-of-line code for AnyValue ---------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+//  This file implements support code for AnyValue.
+//
+//===----------------------------------------------------------------------===//
+
+#include "swift/Basic/AnyValue.h"
+using namespace swift;
+
+AnyValue::HolderBase::~HolderBase() { }
```

Comment:
This is the second definition of this: it's also above in lib/AST/Evaluator.cpp https://github.com/apple/swift/pull/15917/files#diff-dfb37be769b2d50ad4c150591bf84c59R22 ?

Reply:
That one is `AnyRequest::HolderBase::~HolderBase()`.